app/javascript/components/upload/MiscellaneousStep.js

```diff
@@ -13,7 +13,7 @@ const miscFileFilter = file => miscFileTypes.includes(file.file_type)
 
 export default {
   title: 'Miscellaneous / Other',
-  header: 'Documentation &amp; Other Files',
+  header: 'Documentation & Other Files',
```

Comment:
```suggestion
  header: 'Documentation & other files',
```

Reply:
I could be missing something, and please correct me if I'm wrong Devon or @ehanna4, but it looks like this escaped the letter case sweep in #1239.  The purpose / title of the ticket motivation that (SCP-3785) was "Prefer sentence case in upload wizard".   

Whether it's here or in a separate PR before making this GA, please ensure the redesigned upload wizard uses sentence case instead of title case.